### PR TITLE
Fix: replacing deferredLogin by the loginCallback function

### PR DESCRIPTION
**Pull request by soloidx**
Fix of #45 and calling the default callback method

### CODE REVIEW

**Comment by YaroslavG** [comment]
thanks)

File: openfb.js [hunk 1]
```diff
@@ -284,4 +284,4 @@ var openFB = (function () {
         getLoginStatus: getLoginStatus
     }
 
-}());
```
Comment (signalpoint) [inline openfb.js]: I don't think this line should be removed.

**Comment by signalpoint** [comment]
The one line code fix works for me, thank you!

**Comment by ccoenraets** [comment]
Fixed in PR #65

**Pull request closed without merging**